The `sudo make install` / `sudo ninja install` fails because of `slade.pk3` file missing

**SLADE Version**: top of branch `master`
**Operating System**: Linux (Ubuntu 22.04)

**Issue Details**:
Recently, I finally figured how to properly install dependencies and build the thing. However, it fails to be installed because of the `slade.pk3` file is missing. Is that supposed it being no longer generated?

The log I get on attempt to install the thing:
```
$ sudo ninja install
[0/2] Re-checking globbed directories...
[0/1] Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/bin/slade
CMake Error at src/cmake_install.cmake:67 (file):
  file INSTALL cannot find
  "/home/vitaly/_git_repos_other/SLADE/dist/slade.pk3": No such file or
  directory.
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)


FAILED: CMakeFiles/install.util 
cd /home/vitaly/_git_repos_other/SLADE/dist && /usr/local/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
```

--- a/cmake/cmake_uninstall.cmake.in
+++ b/cmake/cmake_uninstall.cmake.in
@@ -7,10 +7,10 @@ string(REGEX REPLACE "\n" ";" files "${files}")
 foreach(file ${files})
   message(STATUS "Uninstalling $ENV{DESTDIR}${file}")
   if(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
-    exec_program(
-      "@CMAKE_COMMAND@" ARGS "-E remove \"$ENV{DESTDIR}${file}\""
+    execute_process(
+      COMMAND "${CMAKE_COMMAND}" -E remove "$ENV{DESTDIR}${file}"
       OUTPUT_VARIABLE rm_out
-      RETURN_VALUE rm_retval
+      RESULT_VARIABLE rm_retval
       )
     if(NOT "${rm_retval}" STREQUAL 0)
       message(FATAL_ERROR "Problem when removing $ENV{DESTDIR}${file}")

--- a/cmake/unix.cmake
+++ b/cmake/unix.cmake
@@ -215,9 +215,11 @@ else(APPLE)
 			RUNTIME DESTINATION bin
 			)
 
+if (BUILD_PK3)
 		install(FILES "${SLADE_OUTPUT_DIR}/slade.pk3"
 			DESTINATION share/slade3
 			)
+endif()
 
 		install(FILES "${PROJECT_SOURCE_DIR}/dist/res/logo_icon.png"
 			DESTINATION share/icons/

--- a/cmake/unix.cmake
+++ b/cmake/unix.cmake
@@ -242,7 +242,7 @@ if(NOT TARGET uninstall)
         IMMEDIATE @ONLY)
 
     add_custom_target(uninstall
-        COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
+        COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake COMMENT "Uninstall the project...")
 endif()
 
 if (NOT NO_COTIRE)

